### PR TITLE
Support alluxiofs:// schema

### PR DESCRIPTION
**Pull request by LuQQiu**

### CODE REVIEW

**Comment by LuQQiu** [comment]
i am modifying the local_fallback fsspec abstract test cases  to increase the test coverage but encounter hard to debug issue when numerous test cases. I am figuring out that part

**Comment by LuQQiu** [comment]
@lucyge2022 @ChunxuTang PTAL, thanks

**Comment by ChunxuTang** [comment]
Quick question: @LuQQiu Will this change have an impact on the customer side? Do you want to update the doc?

**Comment by LuQQiu** [comment]
@ChunxuTang Yeah, thanks for reminding, i am modifying the doc
basically it will change from register("alluxio") to register("alluxiofs")

**Comment by LuQQiu** [comment]
TODO with the schema changes, performance tests needed to be updated

**Comment by LuQQiu** [comment]
@ChunxuTang PTAL, thanks

File: alluxiofs/core.py [hunk 1]
```diff
@@ -145,45 +160,75 @@ def _strip_protocol(path):
...
+
     def fallback_handler(alluxio_impl):
         @wraps(alluxio_impl)
-        def fallback_wrapper(self, path, *args, **kwargs):
-            path = self._strip_alluxio_protocol(path)
+        def fallback_wrapper(self, *args, **kwargs):
+            signature = inspect.signature(alluxio_impl)
+            bound_args = signature.bind(self, *args, **kwargs)
+            bound_args.apply_defaults()
+            for param in ["path", "path1", "path2", "lpath", "rpath"]:
```
Comment (ChunxuTang) [inline alluxiofs/core.py:192]: Could you document why the params are `["path", "path1", "path2", "lpath", "rpath"]`?

**Review by ChunxuTang — APPROVED** [review]
LGTM. Just left some minor comments.

File: alluxiofs/core.py (hunk 1 above)
Comment (LuQQiu) [reply alluxiofs/core.py:192]: yeah, np

File: alluxiofs/core.py [hunk 2]
```diff
@@ -94,11 +96,21 @@ def __init__(
             )
         self.kwargs = target_options or {}
         self.fs = None
+        self.target_protocol = None
         if fs is not None:
             self.fs = fs
+            if isinstance(self.fs.protocol, tuple):
```
[the remark this answers is not in the document]
Comment (LuQQiu) [reply alluxiofs/core.py:102]: e.g. it can be file:// local:// for local filessytem
will do

**Pull request merged**